app/assets/javascripts/uploadcare/utils/messages.coffee

```diff
@@ -12,7 +12,7 @@ uploadcare.namespace 'utils', (ns) ->
     catch
       return
 
-    if message.type of callbacks
+    if message?.type and message.type of callbacks
```

Comment:
`message?.type and message.type` -> `message?.type`

Reply:
@Zmoki nope, because in this case `callbacks` will be iterated anyway, i.e. will be searched for null.